CanvasApi.Client/Admins/AdminApiClient.cs

```diff
@@ -0,0 +1,37 @@
+﻿using CanvasApi.Client._Base;
+using CanvasApi.Client.Admins.Models;
+using CanvasApi.Client.Extentions;
+using System;
+using System.Collections.Generic;
+using System.Net.Http;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace CanvasApi.Client.Admins
+{
+    public class AdminApiClient : ApiClientBase, IAdminApiClient
+    {
+        public AdminApiClient(CanvasApiClient parent) : base(parent) { }
+
+        public async Task<IAdmin> Make(long accountId, Action<IAdminMake> admin) =>
+            await ApiClient.ApiOperation<Admin, IAdminMake>(
+                HttpMethod.Post,
+                $"/api/v1/accounts/{accountId}/admins",
+                admin.GetOptions<IAdminMake, AdminMake>()
+                );
+
+        public async Task<IAdmin> Remove(long accountId, long userId, Action<IAdminRemove> options = null) =>
+            await ApiClient.ApiOperation<Admin, IAdminRemove>(
```

Comment:
I will have to check the documentation, but I think this is actually not removing a user, but demoting a user. If that is the case, maybe call the routine Demote?

Reply:
You are correct, I will make that change